endpoints/orders.yaml

```diff
@@ -12,8 +12,7 @@ security:
 info:
   version: 4.0.9
   title: Orders
-  description: Description text
-  * TODO
+  description: |
```

Comment:
This top level subscription should give a a high level overview of the Orders resource. Something like this: "The Orders resource is exposed by the SGS to provide methods for Create, Cancel, Revise, Acknoledge, and Confirmation of an order. The APIs follow the convention established with the TVB 3.2 flow, most of these flows are asynchronous. Asynchronous requests are delivered by the SGS to Counterparty exposed endpoint URLs, these URLs are set by the Counterparty using the Orders subscription API. The payload is opaque to this specification as it is enacpsulated within the payloadData property and encoded as XML per the TVB 3.2 standard."